src/App.jsx

```diff
@@ -1,33 +1,58 @@
 import './App.scss';
+import { useMemo, useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = useMemo(() => {
+    const preparedMovies = [...moviesFromServer];
+
+    if (query) {
+      return preparedMovies.filter(
```

Comment:
The `if` statement is redundant here. The filter function already performs the necessary filtering based on the truthiness of the provided condition.
```suggestion
    return preparedMovies.filter(
```